src/services/activityDB.js

```diff
@@ -7,6 +7,8 @@ const db = [
     location: 'Siegerland',
     duration: 'short',
     type: 'culture',
+    lat: 50.89735,
+    long: 7.87637,
```

Comment:
I would highly recommend to write them out:
```suggestion
    latitude: 50.89735,
    longitude: 7.87637,
```
That might make it easier for you later as mostly (especially with mapbox) the data is passed as the written out string and it is simpler to know what this is all about :)

Reply:
Updated. Had a quick look into leaflet and actually wanted to take it. So, you would recommend mapbox?

Reply:
Uhm, I am not quite sure, since I haven't worked with either one. Both are fine and good to use 🙂. Sorry for the ubiquity.